complete users guide (#635)
ie3-institute/simona: docs/readthedocs/usersguide.md

## Requirements

> - To run and customize the project you need a **Java Development Kit (JDK)** installation, which you can do directly via IntelliJ IDEA. To do this, press ``ALT + Ctrl + Shift + s`` to open the Project Structure window. Go to ``SDKs``, press on the ``+`` sign in the second column and select ``Download JDK``. Then, choose version 17 from any vendor and click ``download``. Also, make sure version 17 is selected in the Project Tab of the Project Structure window.

Reviewer: ```suggestion
- To run and customise the project you need to install **Java Development Kit (JDK)**, which you can do directly from IntelliJ IDEA. To do this, press ``ALT + Ctrl + Shift + s`` to open the Project Structure window. Go to ``SDKs``, press on the ``+`` sign in the second column and select ``Download JDK``. Then select version 17 from any vendor and click ``download``. Also, make sure version 17 is selected in the Project tab of the Project Structure window.
```

> - To run and customize the project you need a **Java Development Kit (JDK)** installation, which you can do directly via IntelliJ IDEA. To do this, press ``ALT + Ctrl + Shift + s`` to open the Project Structure window. Go to ``SDKs``, press on the ``+`` sign in the second column and select ``Download JDK``. Then, choose version 17 from any vendor and click ``download``. Also, make sure version 17 is selected in the Project Tab of the Project Structure window.

Reviewer: I suggest removing the enumeration here and in the following paragraphs so that it fits uniformly into the text

>     ![](../readthedocs/images/usersguide/jdk-download.png)

Reviewer 2: The images are currently not displayed. It should work if you remove the spaces in front.

> - Additionally, a **Scala Plugin** is required to work with SIMONA. If it is not already installed, a notification will pop up at the top of the editor, which lets you quickly download one.

Reviewer: ```suggestion
- Additionally, the **Scala Plugin** is required to work with SIMONA. If it is not already installed, a notification will pop up at the top of the editor, which lets you quickly download it.
```

![](../readthedocs/images/usersguide/scala-plugin.png)

> Finally, you will also need a **Scala SDK**. Similarly to the Scala Plugin, a notification will let you know if you have not installed one already. To fix this issue, go to ``Setup Scala SDK -> create``, then choose the newest version and download.

Reviewer 2: Didn't know one can just choose the newest version, but it works. Nice!

### Model and Grid Data

Besides a simulation configuration, the specifications of each grid component (e.g. lines, transformers, ...) and system participant (e.g. loads, pv plants, ... ) have to be fed into the simulation.

> Within SIMONA, we use the **PowerSystemDataModel (PSDM)**(https://github.com/ie3-institute/PowerSystemDataModel) for modeling the system components.

Reviewer: The Link is not rendered well. I think this works with `[Text](Link)`

### Model and Grid Data

Besides a simulation configuration, the specifications of each grid component (e.g. lines, transformers, ...) and system participant (e.g. loads, pv plants, ... ) have to be fed into the simulation.

> Within SIMONA, we use the **PowerSystemDataModel (PSDM)** for modeling the system components.

Reviewer: May we should include a link to the git repo of PSDM here as well?

Author: Good idea
